### ISSUE TITLE
hadolint job uploads all of the project sources as artifacts

### ISSUE THREAD

**Issue by liff** [CircleCI-Public/docker-orb#17]
### Orb version

0.5.13

### What happened

Given a `config.yml` like:

```yaml
version: 2.1

orbs:
  docker: circleci/docker@0.5.13

jobs:
  test:
    steps:
      - checkout
      # ... and so on ...

workflows:
  version: 2
  test-and-lint:
    jobs:
      - test
      - docker/hadolint
```

Since the last step of the `hadolint` job is `store_artifacts` the job will start uploading the whole source tree as artifacts. In my specific case there are quite a lot of files in the checked out tree and the upload takes rather a long time.

It is possible to work around this by adding something like `artifacts-path: "/nonexistent"` to the hadolint job settings but I don’t quite understand why storing artifacts is a step in a lint job in the first place?

### Expected behavior

Linting a `Dockerfile` shouldn’t upload the whole source tree as artifacts.

**Comment by apolishch** [comment]
We've just been hit with this. Are there any plans to address this?

**Comment by lokst** [comment]
Apologies for the delay in responding to this. It seems to me there isn't a need to store the artifacts as I don't see any artifacts output from hadolint. Does anyone have objections to removal of the `store_artifacts` step? (cc: @CircleCI-Public/cpeng )

**Comment by wmcnamee-tunein** [comment]
please remove store_artifacts

**Comment by lokst** [comment]
This issue is resolved with `circleci/docker@1.0.0` 🎉  The `artifacts-path` parameter is removed from this version, so it will need to be removed from `.circleci/config.yml` if it's specified in there.
